dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
+    login_type = (['none', '', nil].include? params[:login_type]) ? 'email' : params[:login_type]
```

Comment:
If we got rid of the `'none'` option, we'd be able to just check `.blank?` for '' and nil. This can also happen in the model, with a `before_validation` callback, or by overriding `level_type=` to convert those to `email`.

This could (should) be validated in the model (e.g. `validates_inclusion_of :level_type, in: ...`), and ideally the set of valid level types would be exposed in an array. 

Then we can rely on model validations instead of explicitly checking the fields in the controller, and `return head :bad_request unless section.valid?`.